include/zephyr/net/ieee802154_radio.h

```diff
@@ -340,6 +340,9 @@ struct ieee802154_radio_api {
 	/** Stop the device */
 	int (*stop)(const struct device *dev);
 
+	/** Start continuous carrier wave transmission */
+	int (*continuous_carrier)(const struct device *dev);
```

Comment:
How do you leave this mode?

Plus this should be a separate commit, it's not directly related to OT.

Reply:
To leave this mode redio's state should be changed back to receive (as it is done by calling `otPlatRadioReceive()` in `platformRadioTransmitCarrier()`, which then calls `radio_api->start()`). Alternatively, changing radio's state to sleep (e.g. by `ot diag radio sleep` command) will also stop the transmission.

Moved change to a separate commit.

Reply:
Yeah, but from the 15.4 radio driver perspective, I think it's not clear. IMO it should be documented, that in order to leave this mode one should call `start()` to bring it back into recv state, or `stop()`.

Reply:
Right. Added more info to comment above.